### PR TITLE
Script for coverage when SDL received invalid HMI respond

### PR DESCRIPTION
**Pull request by Aleksandriys**
@OVikhrov @aarotar please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua [hunk 1]
```diff
@@ -0,0 +1,77 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua]: @Aleksandriys, remove duplicated "SDL" in requirement summary

File: test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua]: @Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "GENERIC_ERROR") and not <resultCode_received_from _HMI>

File: test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>.
+-- 3. HMI send invalid response on SDL
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua]: @Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3. and p4(by the way, used p3. twice)should be in expected result.

File: test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  --mobile side: expect SetGlobalProperties response
+  EXPECT_RESPONSE(cid, {success = false, resultCode = "GENERIC_ERROR", info = "Invalid message received from system"})
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/017_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_Invalid_response_on_Mobile.lua]: @Aleksandriys, please avoid redundant empty lines in the end of script

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Pull request merged**